src/scss/utilities/_padding.scss

```diff
@@ -1,11 +1,12 @@
 // e.g ons-u-pl-m
```

Comment:
Think we can remove this line too, after that it looks good

Reply:
removed